src/plotContext.ts

```diff
@@ -181,7 +184,7 @@ export function useAxisContext(
           axisContext[id] = {
             type: axis.scale,
             position: axis.position,
-            scientific: true,
+            tickLabelFormat: toExponential,
```

Comment:
We actually need to add the property to `PlotStateAxis` and use it like this:

```suggestion
            tickLabelFormat: axis.tickLabelFormat,
```